lxd/storage/drivers/driver_lvm.go

```diff
@@ -479,6 +479,10 @@ func (d *lvm) Update(changedConfig map[string]string) error {
 // Mount mounts the storage pool (this does nothing for external LVM pools, but for loopback image
 // LVM pools this creates a loop device).
 func (d *lvm) Mount() (bool, error) {
+	if d.config["lvm.vg_name"] == "" {
+		return false, fmt.Errorf("Cannot mount pool as %q is not specified", "lvm.vg_name")
+	}
```

Comment:
How would this code path ever be used?

As `lvm.vg_name` is a required key, it should never be possible to create or update a storage pool without having it be set, therefore no need to check for it since it's guaranteed to always be set.

Reply:
@stgraber it was there primarily for "belt and braces" checking, to avoid a delay trying to repeatedly mount an empty vg name, as it is possible to remove the `lvm.vg_name` property after it has been added. This is arguably a validation issue in itself, and there is the possibility for a separate bit of work to separate the generation of 'default' config from the `create()` function so that it can be run before the `backendLXD.CreatePool()` function calls `Validate()`. I experimented with it yesterday but it was escalating quickly so wanted to focus on this particular issue ATM.

Reply:
Okay. We can keep it for now. Once we have proper validation for default keys, we'll remove it.